docs/scenarios_authoring/HandlingSchemas.md

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: 11
+---`
+
+# Handling Schemas
+
+## Overview
+
+Nussknacker supports two types of schema:
+* [JSON Schema](https://json-schema.org/) in version: `Draft 7`
+* [Avro Schema](https://avro.apache.org/) in version: `1.11.0`
+
+Avro Schema and JSON Schema are available on [Streaming Source/Sink](/docs/scenarios_authoring/DataSourcesAndSinks.md)
+in contrast to [RequestResponse Source/Sink](/docs/scenarios_authoring/RRDataSourcesAndSinks.md) where available is just only one - JSON Schema.
+
+Nussknacker converts `source schema` to its own static `Typing` information to create hinting on the FE Designer,
+whereas `sink schema` is used to validate output data.
+
+## Source conversion
+Conversion at source to specific type means that behind the scene Nussknacker converts `primitive type` to `logical type`
+consequently, the end-user has access to object's methods, e.g. let's imagine the schema contains the field `foo` with logical type `Date`.
+Originally event contains data with the field `foo` where is stored `Long`. After conversion, field `foo` will be converted to `Date`,
+and the end user on Designer will have access to `#input.foo` with type `Date` and can do `#input.foo.toInstant`.
+
+
+## Avro Schema
+
+### Source supports & conversions
+
+* Primitive types: all
+  * String schema is by default converted to `java.lang.String`. You can change default behavior and convert 
+    String schema to `java.lang.CharSequence` by set env `AVRO_USE_STRING_FOR_STRING_TYPE=false`.
+* Complex types: all
+* Logical types with conversion:
+  * `UUID` => `java.util.UUID`
+  * `Decimal` => `java.math.BigDecimal`
+  * `Date` => `java.time.LocalDate`
+  * `Time (millisecond precision)` => `java.time.LocalTime`
+  * `Time (microsecond precision)` => `java.time.LocalTime`
+  * `Timestamp (millisecond precision)` => `java.time.Instant`
+  * `Timestamp (microsecond precision)` => `java.time.Instant`
+
+[//]: # (Missing support for: Local timestamp millisecond precision, Local timestamp microsecond precision, Duration)
+
+### Sink validation & encoding
+
+In Avro we accept on outputs all compatible types with schemas, and also we allow to pass primitives for logical type: 
+* `java.lang.String` => `UUID`
+* `java.lang.Int`  => `Date`
+* `java.lang.Int` => `Time (millisecond precision)`
+* `java.lang.Long` => `Time (microsecond precision)`
+* `java.lang.Long` => `Timestamp (millisecond precision)`
+* `java.lang.Long` => `Timestamp (microsecond precision)`
+
+If any value will be not the proper logical type (e.g. "str" => UUID) then an exception on runtime will be thrown.
```

Comment:
I don't think it's easy to understand. You mean sth like:
"If runtime value cannot be converted to appropriate value (e.g. "notAUUID" cannot be converted to proper UUID), then an error will be reported during runtime" (don't think we have a notion of exception in user documentation?)